cluster/ci/monitoring/README.md

```diff
@@ -8,11 +8,30 @@ have existed already (they usually come with `openshift-monitoring`).
 
 ## Deploy
 
+The deployment has been integrated into our CI system. The following commands are only for debugging:
+
 ```
 $ make prow-monitoring-pre-steps
 $ make prow-monitoring-deploy
 ```
 
+A successful deploy will spawn a stack of monitoring for prow (similar to what we have for `openshift-monitoring`): _prometheus_, _alertmanager_, and _grafana_.
```

Comment:
In the future we want to push all of this upstream, so be wary of references to OpenShift anything

Reply:
Remove 2 openshift ref.
But there are others, like the route.
I think we need to review this before pushing to upstream.